challenges/linked-list/node.py

```diff
@@ -0,0 +1,7 @@
+class Node:
+    def __init__(self, val, next=None):
+        self.val = val
+        self._next = next
```

Comment:
`next` is a function built into Python, so you should avoid using it as a variable name.